tests/ludwig/modules/test_loss_modules.py

```diff
@@ -144,3 +149,151 @@ def test_huber_loss(
 def test_corn_loss(preds: torch.Tensor, target: torch.Tensor, output: torch.Tensor):
     loss = loss_modules.CORNLoss(CORNLossConfig())
     assert torch.isclose(loss(preds, target), output, rtol=0.0001)
+
+
+def test_dict_class_weights_category():
```

Comment:
Do we have a test of class weights as a list?

Reply:
Ya we have `test_experiment_multiclass_with_class_weights` in `test_experiment.py` which uses list class weights and checks the expected outcome.